src/encoded/types/experiment.py

```diff
@@ -223,6 +223,7 @@ def biosample_summary(self,
                                 biosampleObject.get('post_treatment_time'),
                                 biosampleObject.get('post_treatment_time_units'),
                                 treatment_objects_list,
+                                None,
```

Comment:
Can you explain the purpose of the addition of this line?

Reply:
this line is to eliminate the specification of preservation method in the experiment biosample summary. It was added per my request, trying to keep the experiment level summaries terse.

Reply:
The generate_summary_dictionary method in biosample is changed to have preservation_method as an input parameter. Here in the experiment class, this method is called to display biosample summary in the experiment summary. Therefore, we need to supply a value for the preservation_method parameter here. Idan said that he does not want preservation_method to be displayed in biosample.summary of the experiment, so it is set to 0. I am going to change it to two lines of code: preservation_method=None and then pass preservation_method as a parameter.

Reply:
I added preservation_method in test_biosample_summary.

Reply:
Thank you for clarifying - makes sense (especially with the code update)